ironfish-cli/src/testSetup.ts

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import type { Jest } from '@jest/environment'
+import type { JestExpect } from '@jest/expect'
+import type { Global } from '@jest/types'
+
+declare global {
+  const it: Global.GlobalAdditions['it']
+  const test: Global.GlobalAdditions['test']
+  const fit: Global.GlobalAdditions['fit']
+  const xit: Global.GlobalAdditions['xit']
+  const xtest: Global.GlobalAdditions['xtest']
+  const describe: Global.GlobalAdditions['describe']
+  const xdescribe: Global.GlobalAdditions['xdescribe']
+  const fdescribe: Global.GlobalAdditions['fdescribe']
+  const beforeAll: Global.GlobalAdditions['beforeAll']
+  const beforeEach: Global.GlobalAdditions['beforeEach']
+  const afterEach: Global.GlobalAdditions['afterEach']
+  const afterAll: Global.GlobalAdditions['afterAll']
```

Comment:
If any TypeScript expert knows how to put everything from `Global.GlobalAdditions` into the global namespace with 1 line, please let me know!

Reply:
```ts
const {
  it, 
  test, 
  fit, 
  xit, 
  xtest, 
  describe, 
  xdescribe, 
  fdescribe, 
  beforeAll, 
  beforeEach, 
  afterEach, 
  afterAll,
}: Global.GlobalAdditions;
```

Does this work for what you want?

Reply:
That's better, thanks. What I would really like to do is avoiding writing `it`, `test`, `fit`, `xit`, ... I would like to do something like `import * from Global.GlobalAdditions`

Reply:
I don't think TS has a way to import all properties from a type into the global namespace, but maybe someone else knows a better way.